docs/SpatialEpigenome.rst

```diff
@@ -386,12 +367,17 @@ and assign the resulting list of motifs to the variable motifs. We do this to cr
 Apply addImputeWeights to the input Seurat object and assign the result to the variable proj_in_tissue. This step is done to improve the accuracy of the marker features by imputing missing values.::
 
    proj_in_tissue <- addImputeWeights(proj_in_tissue)
+   
+Source the getDeviation_ArchR.R, getGeneScore_ArchR.R, SpatialPlot_new.R, and SpatialDimPlot_new.R script. The source function reads in these scripts that contain R code for specific functions and visualizations to be used in our current R session. ::
+   
+   source(paste0(script.dir, "/", "getDeviation_ArchR.R"))
+   source(paste0(script.dir, "/", "getGeneScore_ArchR.R"))
+   source(paste0(script.dir, "/", "SpatialPlot_new.R"))
+   source(paste0(script.dir, "/", "SpatialDimPlot_new.R"))
 
 Deviation scores and matrices
 ----------------------------------------
-
-Apply getDeviation_ArchR to the modified Seurat object and the list of motifs, along with the result of applying the getImputeWeights function to the 
-modified Seurat object. Assign the result to the variable dev_score. ::
+Compute deviation scores for the motifs of interest using the getDeviation_ArchR() function. The function takes the ArchR project object, the list of motifs, and the imputed weights computed with getImputeWeights(). Assign the resulting deviation scores to the variable dev_scores. ::
```

Comment:
This functions gets the deviation  scorematrix from the archR project. Deviation scores are computed before.